src/Build/Construction/ProjectRootElement.cs

```diff
@@ -619,6 +619,8 @@ public int Version
         /// </summary>
         public DateTime LastWriteTimeWhenRead => Link != null ? RootLink.LastWriteTimeWhenRead : _lastWriteTimeWhenRead;
 
+        internal DateTime? StreamTime = null;
```

Comment:
Seems that some DateTime's in this type are stored as Utc and some are not. Would it be helpful to consistently use UTC? Or else maybe consider a suffix on the field name eg `NotUtc`

Reply:
It's important for StreamTime to be in the same time zone as the file's time and _lastWriteTimeWhenRead, and I think using local time is the least confusing. I changed it to explicitly convert it to local time just for clarity.

Reply:
I prefer the other direction: use UTC everywhere. That's what we do for most things. Unfortunately, we didn't catch this when we added `LastWriteTimeWhenRead` as a public field instead of `LastWriteTimeWhenReadUtc`. So I guess converging on local time in this class is probably the way to go.

Reply:
The public properties need not have the Utc suffix, but the backing fields could be Utc, and have utc suffix, without breaking anyone (unless the machine is changing timezones)